### ISSUE TITLE
`map_dep()` by locationID rather than deploymentID?

### ISSUE THREAD

**Issue by Rafnuss** [inbo/camtraptor#354]
In my dataset, I often have multiple deployments at the same location. Some feature (such as n_species) would look a lot better if the plot was by location rather than deployment. If you think that it's a good idea to offer this option (I couldn't find it in the issues), I'm happy to have a go at drafting a PR. Just wanted to know your opinion before.

**Comment by damianooldoni** [comment]
Indeed, this is something some colleagues/users at INBO are suggesting as well. Thanks @Rafnuss for mentioning it in a issue.

What if lat/lon is not the same for all deployments at the same location? Some options:
1. Use the coordinates of the centroid: it means loading sf package as dependency to calculate it, not really a very big fan of doing so.
2. Use the coordinate of the first deployment. Why the first one? Just easier...

In both cases, I think we should return a warning if such situation occurs.

@Rafnuss, @MartijnUH, @jimcasaer: any thoughts about it?

**Comment by Rafnuss** [comment]
Is it allow to have different lat, lon for the same locationID? Seems like this should not be allow at all (hence warning seems appropriate). Why would you need `sf` for a centroid? Average lat and average lon should do the job? I would actually prefer a median such that the location actually corresponds to one possibility rather than somewhere in the middler of nowhere or in case of outlier position.

**Comment by jimcasaer** [comment]
When cameras are replaced in the same grid of the monitoring frame they sometimes have the same locationname and therefore the same locationID, however the coordinates can change. Until now we each time gave a new locationname if the coordinates were not exactly the same and subsequently deduced the gridcell from the locationname. E.G locationname == Gridcel_array_year

**Comment by damianooldoni** [comment]
DIscussing with @peterdesmet: probably the name `map_summary()` is better as this function will _map_ both the summaries created respectively by `summarise_observations()` and `summarise_deployments()`.

Do we need a default feature/measurement? Probably the first one, i.e. the first column not belonging to the grouping. This means:
- `n_scientificName` for `summarise_observations() %>% map_summary()`
- `effort_duration` for `summarise_deployments() %>% map_summary()`

**Comment by damianooldoni** [comment]
Solved in #373. Not yet in `main`, but in `version1.0` branch.
